x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/form/validations.ts

```diff
@@ -89,15 +89,28 @@ export const rolloverThresholdsValidator: ValidationFunc = ({ form, path }) => {
   }
 };
 
-export const minAgeValidator: ValidationFunc<any, any, any> = (arg) =>
-  numberGreaterThanField({
-    than: 0,
-    allowEquality: true,
-    message: i18nTexts.editPolicy.errors.nonNegativeNumberRequired,
-  })({
-    ...arg,
-    value: arg.value === '' ? -Infinity : parseInt(arg.value, 10),
-  });
+export const integerValidator: ValidationFunc<FormInternal, string, string> = (arg) => {
```

Comment:
@jloleysens Do you mean like this? 
```
export const integerValidator: ValidationConfig<FormInternal, string, string> = {
  validator: (arg) => {
    if (!Number.isInteger(Number(arg.value ?? ''))) {
      return { message: i18nTexts.editPolicy.errors.integerRequired };
    }
  },
};
```
and then use it like this
```
export const minAgeValidator: Array<ValidationConfig<FormInternal, string, string>> = [
  {
    validator: (arg) =>
      numberGreaterThanField({
        than: 0,
        allowEquality: true,
        message: i18nTexts.editPolicy.errors.nonNegativeNumberRequired,
      })({
        ...arg,
        value: arg.value === '' ? -Infinity : parseInt(arg.value, 10),
      }),
  },
  integerValidator,
];
```

Reply:
I guess I was thinking it would be best the way that the `minAgeValidator` was originally:

```ts
export const minAgeValidator: ValidationFunc<any, any, any> = (arg) =>
  numberGreaterThanField({
    than: 0,
    allowEquality: true,
    message: i18nTexts.editPolicy.errors.nonNegativeNumberRequired,
  })({
    ...arg,
    value: arg.value === '' ? -Infinity : parseInt(arg.value, 10),
  });
```

But more importantly I think `integerValidator` and `minAgeValidator` should be the same :)

Reply:
Ah I see @jloleysens! I haven't changed the `minAgeValidator` directly because I reuse `integerValidator` for rollover max age and max documents. Those fields can be zero but min age can't be zero, that's why 2 different validators.